DelvUI/Plugin.cs

```diff
@@ -187,29 +171,17 @@ private void Draw()
                          || _pluginInterface.ClientState.Condition[ConditionFlag.BetweenAreas51];
```

Comment:
Replace `_pluginInterface` with singleton reference

Reply:
No need to do it in the class that has the actual reference.